cypress/integration/mocked/onboarding_flow.1.cy.ts

```diff
@@ -297,6 +298,219 @@ describe('Onboarding flow', () => {
 				.should('have.attr', 'href')
 				.and('include', decodeURIComponent(returnUrl));
 		});
+		//@AB_TEST: 3 Stage Registration Flow Test START
+		describe('AB Test 3 stage registration flow', () => {
```

Comment:
A mocked test is great, but we generally try to use end to end tests now, so an `ete-okta` test will be better as that will test real APIs, you can use `setMvtId` within the end to end tests too. The mocked test is still helpful though.